src/lib/deploy.js

```diff
@@ -268,6 +346,49 @@ function chooseSmartContract(config, deploy, network) {
   return '';
 }
 
+/**
+ * Find the file name of the smart contract to be deployed.
+ * @param {string}    buildPath    The glob pattern--e.g. `build/**\/*.js`
+ * @param {string}    contractName The user-specified contract name to deploy.
+ * @returns {string}  The file name of the user-specified smart contract.
+ */
+async function findSmartContractToDeploy(buildPath, contractName) {
+  const files = await glob(buildPath);
+  const re = new RegExp(`class ${contractName} extends SmartContract`, 'gi');
+  for (const file of files) {
+    const contract = fs.readFileSync(file, 'utf-8');
+    if (re.test(contract)) {
+      return path.basename(file);
+    }
+  }
+}
+
+/**
+ * Returns a field element from the specified string.
+ * @param {string} s The string to convert to a Field element
+ * @returns A Field element
+ */
+function toFieldFromString(s) {
+  let bits = [];
+  for (let i = 0; i < s.length; ++i) {
+    const c = s.charCodeAt(i);
+    for (let j = 0; j < 8; ++j) {
+      bits.push(((c >> j) & 1) === 1);
+    }
+  }
+  return Field.ofBits(bits);
+}
+
+/**
+ * Returns the specified private key in a format that can be used by the `mina-signer`
+ * @param {string} privateKey A private key created by SnarkyJS
+ * @returns A private key that can be used by the mina-signer
```

Comment:
this doc comment sounds like the opposite of what the function is doing :D

Reply:
Yikes, that was a brain fart! Will fix this, good catch!